liiatools_pipeline/ops/common_la.py

```diff
@@ -240,15 +240,21 @@ def create_concatenated_view(current: DataframeArchive, config: CleanConfig):
         f"concatenated/{config.dataset}", recreate=True
     )
     existing_files = concat_folder.listdir("/")
+    log.info(f"Existing files: {existing_files}")
 
     for la_code in authorities.codes:
         la_files_regex = f"{la_code}_{config.dataset}_"
+        log.info(f"Removing files with regex: {la_files_regex}")
         pl.remove_files(la_files_regex, existing_files, concat_folder)
+        log.info(f"Successfully removed files")
 
         if config.dataset == "annex_a":
+            log.info(f"Concatenating annex a data for {la_code}")
             concat_data = current.current(la_code, deduplicate_mode="N")
+            log.info(f"Data concatenated: {concat_data}")
```

Comment:
I don't think we should have this, as it will actually post row-level data

Reply:
Have now removed